src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -10,11 +10,7 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import seedu.address.commons.exceptions.IllegalValueException;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.person.*;
```

Comment:
Don't use this import syntax. Separate them out by disabling Intellij auto-optimise feature for imports.